pkg/plugins/plugins.go

```diff
@@ -89,21 +98,34 @@ func (p PluginDTO) IsSecretsManager() bool {
 	return p.JSONData.Type == SecretsManager
 }
 
-func (p PluginDTO) IncludedInSignature(file string) bool {
-	// permit Core plugin files
-	if p.IsCorePlugin() {
-		return true
+type File interface {
+	fs.File
+	io.Seeker
```

Comment:
> https://pkg.go.dev/io/fs#File: A file may implement io.ReaderAt or io.Seeker as optimizations.

Does that imply we should try cast to `fs.File` to `io.Seeker` whenever needed? :thinking: 

What is it that requires an `io.Seeker`, the plugin assets handler?

Reply:
Exactly yes. I was unsure what this comment meant - so if it means I can type cast then I can just is fs.File